weldx/asdf/tags/weldx/core/common_types.py

```diff
@@ -145,6 +145,8 @@ def to_tree(cls, node: Variable, ctx):
             data = node.data
         dtype = node.data.dtype.str
         data = cls.convert_time_dtypes(data=data)
+        if not data.shape:  # scalar
```

Comment:
So data always has a shape information attached, if it is a scalar? Is this yet covered by a test?

Reply:
sorry, I haven't checked the coverage report. Thanks for fixing this.

Reply:
data should always be a `np.ndarray` here, just need to catch scalar values for the correct ASDF serialization